doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -70,6 +59,17 @@ \subsection{[meta]}
 \item {\em example:} \lstinline=http://suites/${CYLC_SUITE_NAME}/index.html=
 \end{myitemize}
 
+\subsubsection{group}{ [meta] \textrightarrow group}
+
+A single line group name for a suite. It is displayed in the gscan
+window and can be used as a way of grouping related suites together.
+\lstinline=cylc show= command.
```

Comment:
Should this say "...with the `cylc show` command? (It was in current docs on master too)

Reply:
I think this was a typo as `cylc show` doesn't know about groups - they're only used by gscan, right? ... in which case maybe this should read: *A group name for a suite. In the gscan GUI, suites with the same group name can be collapsed into a single state summary when the "group" column is displayed.*